File: BE/main_service/model/vote_rating.go
package model

type VoteRating struct {
	ID            string `json:"id,omitempty"`
	RatingID      string `json:"rating_id,omitempty"`
	UserID        string `json:"user_id,omitempty"`
	Like          int    `json:"like,omitempty"`
	Stake         int    `json:"stake,omitempty"`
	TransactionID string `json:"transaction_id,omitempty"`
}

File: syncer/model/voteRating.js
const Sequelize = require('sequelize');
const sequelize = require('../config/pg')
var VoteRating = sequelize.define("vote_rating", {
    id: {
        type: Sequelize.BIGINT,
        primaryKey: true,
    },
    ratingId: {
        type: Sequelize.BIGINT,
    },
    userId: {
        type: Sequelize.BIGINT,
    },
    like: {
        type: Sequelize.INTEGER,
    },
    stake: {
        type: Sequelize.INTEGER,
    },
    transactionId: {
        type: Sequelize.TEXT,
    },
})

module.exports = VoteRating